jobs/datadog-cc-worker/templates/process.yaml.erb

```diff
@@ -0,0 +1,6 @@
+init_config:
+
+instances:
+  - name: cc api worker
```

Comment:
There [is other option in the case of `cloud_controller_worker` job](https://github.com/cloudfoundry/cloud_controller_ng/blob/10acca68afc6a166fa77420c3c4799ca5ccc10e2/bosh/jobs/cloud_controller_worker/spec#L487-L489) which defaults to 1.